go-controller/pkg/ovn/ovn.go

```diff
@@ -119,9 +142,153 @@ func (oc *Controller) Run() error {
 			return err
 		}
 	}
+
+	if config.EmptyLbEvents || config.Kubernetes.OVNEmptyLbEvents {
```

Comment:
@putnopvut once you've consolidated the config stuff, you can just use `if config.Kubernetes.OVNEmptyLbEvents {` here.